src/Microsoft.ComponentDetection.Detectors/pip/Contracts/PipInstallationReport.cs

```diff
@@ -0,0 +1,34 @@
+namespace Microsoft.ComponentDetection.Detectors.Pip;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+/// <summary>
+/// See https://pip.pypa.io/en/stable/reference/installation-report/#specification.
+/// </summary>
+public class PipInstallationReport
+{
+    /// <summary>
+    /// Version of the installation report specification. Currently 1, but will be incremented if the format changes.
+    /// </summary>
+    [JsonProperty("version")]
+    public int Version { get; set; }
```

Comment:
[The documentation][1] says this is a string
>version: the string 1.

[1]: https://pip.pypa.io/en/stable/reference/installation-report/#specification

Reply:
Changed it to a string and now compare using `Version` instead